Adopt NumFOCUS' Code of Conduct (#2)
scipy-india/.github: CODE_OF_CONDUCT.md

# SciPy India Code of Conduct

SciPy India is a multidisciplinary community and forum for users and developers of the SciPy stack in India. Our community is built on the principles of collaboration, inclusivity, and respect for all participants. We strive to create a community where everyone feels included and valued. We are committed to providing a welcoming and inspiring community for all, regardless of background, identity, or experience. We expect that all participants treat each other with dignity and kindness.

The SciPy India community has adopted and strictly follows the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct). This document outlines our expectations for participants within the community and provides a mechanism for reporting unacceptable behaviour and addressing grievances that may arise.

Everyone is expected to follow the Code of Conduct in all community spaces, including online or physical fora (GitHub, Zulip, and more), meetups, events, workshops, conferences, and any other in-person interactions between members at SciPy India gatherings.

## Reporting

> If you would like to report a violation, please report it via our [anonymous reporting form](TODO: link to reporting form here). We will get to it as soon as possible. Please do not hesitate to report any behaviour that you believe violates the Code of Conduct, even if you are unsure whether it constitutes a violation.

Reviewer: it's almost same as the nf reporting form
```suggestion
If you would like to report a violation, please report it via our [anonymous reporting form](https://form.jotform.com/scipyindia/scipy-india-coc-reporting-form). We will get to it as soon as possible. Please do not hesitate to report any behaviour that you believe violates the Code of Conduct, even if you are unsure whether it constitutes a violation.
```

Reviewer: does thumbs up mean I should go ahead and merge it?

Author: Ah, I sent a message internally just now!

Reviewer: all comments are on the editing inside the form-- so I'll merge it.

Reviewer: Sorry didn't commit the suggestion; link added in https://github.com/scipy-india/.github/commit/87e99bd66e076482c5c757c7613f27abfd7a749c

# SciPy India Code of Conduct

> SciPy India is a multidisciplinary community and forum for users and developers of the SciPy stack in India. Our community is built on the principles of collaboration, inclusivity, and respect for all participants. We strive to create an environment where everyone feels welcome and valued. We are committed to providing a welcoming and inspiring community for all, regardless of background, identity, or experience. We expect that all participants treat each other with respect and kindness.

Reviewer: ```suggestion
SciPy India is a multidisciplinary community and forum for users and developers of the SciPy stack in India. Our community is built on the principles of collaboration, inclusivity, and respect for all participants. We strive to create a community where everyone feels included and valued. We are committed to providing a welcoming and inspiring community for all, regardless of background, identity, or experience. We expect that all participants treat each other with dignity and kindness.
```

> The SciPy India community is governed by the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct). This document outlines our expectations for participants within the community and provides a mechanism for reporting unacceptable behaviour and addressing grievances that may arise.

Reviewer: "governed by" might sounds like NumFOCUS is SciPy india's parent org :)
```suggestion
The SciPy India community has adopted and strictly follows the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct). This document outlines our expectations for participants within the community and provides a mechanism for reporting unacceptable behaviour and addressing grievances that may arise.
```

Author: Applied!

> sounds like NumFOCUS is SciPy india's parent org :)

It will be sometime soon, hopefully :)

> Everyone is expected to follow the Code of Conduct in all community spaces, including online or physical fora, meetups, events, workshops, conferences, and any other interactions related to SciPy India.

Reviewer: could we also add zulip, github and also any other in-person interactions one might have with another SciPy community member?

Author: Yes, Zulip and GitHub are covered under online fora. I'll mention both of them explicitly.

Author: > also any other in-person interactions one might have with another SciPy community member?

Agreed!

## Reporting

> If you would like to report a violation, please do so by contacting the SciPy India team or through our [anonymous reporting form](TODO: link to reporting form here) . Please do not hesitate to report any behaviour that you believe violates the Code of Conduct, even if you are unsure whether it constitutes one.

Reviewer: We don't have a scipy india email yet -- so I've put our emails(aditijuneja7@gmail.com , agriyakhetarpal@outlook.com) in the form for now. That's what I've been doing everywhere else so far. Lmkwyt.

Also, we can consider putting the report form link in the readme aswell.

```suggestion
If you would like to report a violation, please repot it via our [anonymous reporting form](TODO: link to reporting form here) . And we will get to resolving the report ASAP. Please do not hesitate to report any behaviour that you believe violates the Code of Conduct, even if you are unsure whether it constitutes one.
```

Author: > We don't have a scipy india email yet -- so I've put our emails([aditijuneja7@gmail.com](mailto:aditijuneja7@gmail.com) , [agriyakhetarpal@outlook.com](mailto:agriyakhetarpal@outlook.com)) in the form for now. That's what I've been doing everywhere else so far. Lmkwyt.

Yes, sounds good to me. The only downside is that users might not be comfortable reporting something that concerns either of us. Let's track this in a follow-up issue.

Author: Applied this suggestion with some slight edits.

When reporting, please provide as much detail as possible, including:

- A description of the incident
- The names of any individuals involved
- The date and time of the incident
- Any witnesses or evidence (e.g., screenshots, links)

> - Your contact information (should you wish to be contacted for following up)

Reviewer: All this will be in the form -- so we can remove it also. lmkwyt.

## Enforcement

> A dedicated team will review all reports and take appropriate action, which may include warnings, temporary or permanent bans from community spaces, or other measures deemed necessary to ensure a safe and welcoming environment for all participants.

Reviewer: ```suggestion
A dedicated team will review all reports and take appropriate action, as per the NumFOCUS's CoC, which may include warnings, temporary or permanent bans from community spaces, or other measures deemed necessary to ensure a safe and welcoming environment for all participants.
```

## Contact

> If you have any questions, concerns, or suggestions regarding the Code of Conduct, please reach out to the community on [Zulip](https://scipyindia.zulipchat.com) or via email at[mailto:TODO]

Reviewer: I think we can put our email here for now.
